### DISCUSSION TITLE
schema adjustments, running local mangal

### DISCUSSION THREAD

**Discussion by gottacatchenall** [mangal-interactions/contribute#5] [Ideas]
As a continuation of [this](https://github.com/EcoJulia/Mangal.jl/issues/23) it looks like running a local version of Mangal would be useful as part of an ongoing project. 

Couple things:

1) In the future it may make sense to adjust the schema to enable many-to-many relationship between attributes and interactions. Not sure how much annoyance this will cause to rebuild the db 

2) `mangal-api` seems to rely on deprecated versions of `webpack` versions (<4) 

3) @tpoisot mentioned in the above issue that `mangal-api` also might be older than the current production mangal stack? possibly related to (2) 

4) How is data typically added to the database? Is there a template of a programmatic ways to go from CSV to database? I couldn't find `POST` slugs in `mangal-api`. For this ongoing project an easy way to add data to the local version of mangal would be nice 

5) running `mangal-vagrant` fails when setting env variables, not sure what they should be set to

**Comment by tpoisot** [comment]
All but 4 are going to be discussed next week as part of our maintenance meeting

For point 4, there's a secret package for upload 🕵️ - at this point we'd rather do the upload ourselves, because we're using this to iron out some bugs

**Comment by gottacatchenall** [comment]
could i get access to this :detective: repo to quickly add bumblebee interaction CSVs into a local version of mangal?

**Reply by gottacatchenall** [reply]
bumping this @tpoi as we will soon have access to this data
